pkg/tuned/controller.go

```diff
@@ -350,11 +354,17 @@ func (c *Controller) sync(key wqKeyKube) error {
 		change.provider = profile.Spec.Config.ProviderName
 		change.recommendedProfile = profile.Spec.Config.TunedProfile
 		change.debug = profile.Spec.Config.Debug
+		err = util.SetLogLevel(profile.Spec.Config.Verbosity)
+		if err != nil {
+			klog.Errorf("failed to set log level %d: %v", profile.Spec.Config.Verbosity, err)
+		} else {
+			klog.Infof("set log level %d", profile.Spec.Config.Verbosity)
+		}
```

Comment:
this hunk depends on a unbackported PR: #1114 . We shold either also backport it (probably not the best approach) or remove this hunk in a separate commit in this PR.

Reply:
Honestly, I tend to lean towards the latter.  Dropping this hunk.